tests/jobs/localisation/test_config1.py

```diff
@@ -0,0 +1,831 @@
+                "PARAM_NODE3:PARAM4",
+                "PARAM_NODE1X:X*",
+            ],
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE3:PARAM4",
+                "PARAM_NODE1X:X1",
+                "PARAM_NODE1X:X2",
+                "PARAM_NODE1X:X3",
+            ],
+        ),
+        (
+            ["PARAM_NODE3:P*", "PARAM_NODE2*:*2*"],
+            [
+                "PARAM_NODE2:PARAM2",
+                "PARAM_NODE22:P2",
+                "PARAM_NODE22:P22",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE3:PARAM4",
+            ],
+        ),
+        (
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:*",
+                "PARAM_NODE1X:X3",
+                "PARAM_NODE1X:X1",
+            ],
+            [
+                "PARAM_NODE1:PARAM1",
+                "PARAM_NODE1X:X1",
+                "PARAM_NODE1X:X3",
+                "PARAM_NODE2:PARAM3",
+                "PARAM_NODE2Y:Y1",
+                "PARAM_NODE2Y:Y2",
+                "PARAM_NODE2Y:Y3",
+            ],
+        ),
+        (
+            "PARAM_FIELD*",
+            ["PARAM_FIELD1", "PARAM_FIELD2", "PARAM_FIELD3"],
+        ),
+        (
+            "PARAM_FIELD1",
+            ["PARAM_FIELD1"],
+        ),
+        (
+            "PARAM_GEN:*",
+            ["PARAM_GEN:0", "PARAM_GEN:1", "PARAM_GEN:2", "PARAM_GEN:3", "PARAM_GEN:4"],
+        ),
+    ],
+)
+def test_simple_config(param_group_add, expected):
+    data = {
+        "log_level": 2,
+        "correlations": [
+            {
+                "name": "some_name",
+                "obs_group": {"add": ["OBS1"]},
+                "param_group": {
+                    "add": param_group_add,
+                },
+            }
+        ],
+    }
+    conf = LocalisationConfig(observations=ERT_OBS, parameters=ERT_PARAM, **data)
+    assert len(conf.correlations) == 1
+    assert conf.correlations[0].obs_group.result_items == ["OBS1"]
+    assert sorted(conf.correlations[0].param_group.result_items) == sorted(expected)
+
+
+@pytest.mark.parametrize(
+    "obs_group_add, param_group_add,  param_group_remove, expected_error",
+    [
+        (
+            "OBS1",
+            "PARAM_NODE1",
+            "[]",
+            "No match for: PARAM_NODE1",
+        ),
+        (
+            ["OBS1", "OBS2"],
+            ["PY*:*", "PARAM_FILED*"],
+            "[]",
+            "No match for: PY*:*",
+        ),
+        (
+            ["OBS2*"],
+            ["PARAM_FIELD*"],
+            ["PARAM_FIELD2:*"],
+            "No match for: PARAM_FIELD2:*",
+        ),
+        (
+            ["OBS*"],
+            "PARAM_NODE1:PARAM1",
+            "P*:*:*",
+            "No match for: P*:*:*",
+        ),
+        (
+            [],
+            "PARAM_NODE1:PARAM1",
+            [],
+            "",
+        ),
+    ],
```

Comment:
Same comment as above, are these all testing something unique?

Reply:
I think these tests are relevant. They test misspelling  like the P*;*:* and tests the 'remove' part of the specification.